database/mongoDatabase.go

```diff
@@ -29,10 +33,19 @@ func (mongoDB *MongoDB) GetClubs() ([]dao.Club, error) {
 	return clubs, nil
 }
 
-func NewMongoDB(client *mongo.Client) *MongoDB {
-	return &MongoDB{client}
+func (mongoDB *MongoDB) PutRating(rating dao.Rating) error {
+	collection := mongoDB.client.Database("mobile-app").Collection("ratings")
+
+	filter := bson.M{"clubId": rating.ClubID, "userId": rating.UserID}
```

Comment:
Make "clubId" and "userId" const strings, or utilise the bson properties in the Rating struct